### PR TITLE
Release notes for SmartSim 0.4.1

### PR DESCRIPTION
**Pull request by ashao**

### CODE REVIEW

**Comment by Spartee** [comment]
Congrats y'all!! This release looks great!!

File: doc/changelog.rst [hunk 1]
```diff
@@ -11,6 +11,71 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
...
+
+Released on June 24, 2022
+
+Description:
+This release of SmartSim introduces a new experimental feature to help make
+SmartSim workflows more portable: the ability to run simulations models in a
+container via Singularity. This feature has been tested on a small number of
+platforms and we encourage users to provide feedback on its use.
+
+We have also made incremental improvements in a variety of areas: being able to
```
Comment (mellis13) [inline doc/changelog.rst:25]: Maybe remove "incremental"?

File: doc/changelog.rst [hunk 2]
```diff
@@ -11,6 +11,71 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
...
+load scripts and machine learning models into the database directly from
+SmartSim driver scripts, the ability to use `KeyDB` instead of `redis-server`,
```
Comment (mellis13) [inline doc/changelog.rst:27]: Maybe phrase as "the ability to choose `KeyDB` or `Redis` during SmartSim installation"

File: doc/changelog.rst [hunk 3]
```diff
@@ -11,6 +11,71 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
...
+and support for using OpenMPI to launch workflows. Reducing the barrier to entry
```
Comment (mellis13) [inline doc/changelog.rst:28]: Should we say "RunSettings" instead of "to launch workflows"?

File: doc/changelog.rst [hunk 4]
```diff
@@ -11,6 +11,71 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
...
+Launcher improvements
+
+    - New methods for RunSettings (_SmartSim-PR166) (_SmartSim-PR170)
+    - Experimental: add support for running models via Singularity (_SmartSim-PR204)
+
+Documentation and tutorials
+
+    - Tutorial updates (_SmartSim-PR155) (_SmartSim-PR203) (_SmartSim-PR208)
+    - Add SmartSim Zoo info to documentation (_SmartSim-PR175)
+    - New tutorial for demonstrating online training(_SmartSim-PR176)
```
Comment (mellis13) [inline doc/changelog.rst:44]: Missing space before "("

File: doc/changelog.rst [hunk 5]
```diff
@@ -11,6 +11,71 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
...
+    - Container now available for tutorials (_SmartSim-PR188)
+
+Quality of life enhancements and key bug fixes
```
Comment (mellis13) [inline doc/changelog.rst:47]: I think before we just called this section "General Improvements and Bug Fixes:".  Maybe we stick with that?

File: doc/changelog.rst [hunk 6]
```diff
@@ -11,6 +11,71 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
...
+
+    - Set models and scripts at the driver level (_SmartSim-PR185)
+    - Optionally use KeyDB for the orchestrator (_SmartSim-PR180)
+    - Ability to specify system-level libraries (_SmartSim-PR154) (_SmartSim-PR182)
+    - Wrong handling of LSF gpus_per_shard (_SmartSim-PR164)
```
Comment (mellis13) [inline doc/changelog.rst:52]: Maybe prepend this line with "Fix"

File: doc/changelog.rst [hunk 7]
```diff
@@ -11,6 +11,71 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
...
+    - Fix for bug with re-running smart build (_SmartSim-PR165)
```
Comment (mellis13) [inline doc/changelog.rst:53]: Maybe phrase "Fix error when re-running smart build"

File: doc/changelog.rst [hunk 8]
```diff
@@ -11,6 +11,71 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
...
+    - Generator no longer hangs when tagged configuration variables are missing (_SmartSim-PR177)
```
Comment (mellis13) [inline doc/changelog.rst:54]: Maybe rephrase "Fix generator hanging when ..."

File: doc/changelog.rst [hunk 9]
```diff
@@ -11,6 +11,71 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
...
+and support for using OpenMPI to launch workflows. Reducing the barrier to entry
+by providing more extensive tutorials and improving the documentation was
+another key focus of this release cycle. Interested newcomers and users to
+SmartSim can now download a docker image to run many of the tutorials described
+in the documentation.
+
+
+Launcher improvements
+
+    - New methods for RunSettings (_SmartSim-PR166) (_SmartSim-PR170)
```
Comment (mellis13) [inline doc/changelog.rst:37]: Should we say "OpenMPI RunSettings"

**Review by mellis13 — APPROVED** [review]
Thanks for putting this draft together -- really excited to see 0.4.1 get released!

File: doc/changelog.rst (hunk 2 above)
Comment (ashao) [reply doc/changelog.rst:27]: Slightly reworded from your suggestion to "install-time choice to use either `KeyDB` or `Redis` for the Orchestrator"

File: doc/changelog.rst (hunk 9 above)
Comment (ashao) [reply doc/changelog.rst:37]: I think it's more accurate to say "including OpenMPI"?

Comment (ashao) [reply doc/changelog.rst:37]: Oh wait, I think these PRs are really just the additional setter methods for generic RunSettings. However, I should include the OpenMPI-related PRs.

**Comment by al-rigazzi** [comment]
We switched to a different PR to be able to edit the branch. Thanks @ashao for the work!
